src/plugins/tune/tune_impl.cpp

```diff
@@ -113,38 +114,29 @@ void TuneImpl::play_tune_async(
 
     LogDebug() << "About to send tune: " << tune_str;
 
-    if (tune_str.size() > 230) {
-        report_tune_result(callback, Tune::Result::TUNE_TOO_LONG);
+    if (tune_str.size() > 248) {
```

Comment:
This could be `MAVLINK_MSG_PLAY_TUNE_V2_FIELD_TUNE_LEN`

Reply:
Thanks!